### ISSUE TITLE
Blank screen on startup in Win7

### ISSUE THREAD

**Issue by JedzieGoat** [SabakiHQ/Sabaki#766]
When I start Sabaki v.0.51.1 on Windows 7, it shows a blank white screen on the desktop, which later changes to a blank black screen.  When I mouse over near the top edge of the window, it shows dropdown menus, which actually seem to work but the screen stays blank.  "New" seems to generate a new game - I can click somewhere on the blank screen and hear sounds like stone placements.  I suppose that I could play a blindfold game this way, but is that even a thing in Go?  When I click "New Window", it creates another - blank -window.  I have tried the install version and the portable version with the same results.  Any help would be appreciated, thanks.

![SabakiBlank](https://user-images.githubusercontent.com/77457141/104629885-b5fbec00-565f-11eb-8f41-04a8772c51c9.jpg)

![SabakiBlank2](https://user-images.githubusercontent.com/77457141/104629931-c44a0800-565f-11eb-87ee-5d03d08d5615.jpg)

![SabakiBlank3](https://user-images.githubusercontent.com/77457141/104629952-cb711600-565f-11eb-83f3-1d7c916b5534.jpg)

**Comment by cryptsport** [comment]
I have been using different versions of Sabaki, Windows 7 64 for several years, everything is fine

**Comment by JedzieGoat** [comment]
@cryptsport: I'm glad that it works on YOUR machine, but I posted a problem with MY machine...

**Comment by cryptsport** [comment]
I wrote this to make it clear that this failure is not typical for Windows7. on the other hand, you have not posted any information about your hardware - video card, processor, memory, Windows version, video driver, log file. in some cases, you can immediately identify an error from this information

**Comment by JedzieGoat** [comment]
Your comment was clear enough, but as a statement, unhelpful.
The suggestions in your recent comment ARE helpful, thanks,
and here's the info, per DxDiag 6.01.7601.17514 64bit Unicode:

Lenovo b570 (LapTop)
       System Model: 1068 APU
               BIOS: Default System BIOS
          Processor: Intel(R) Pentium(R) CPU B950 @ 2.10GHz (2 CPUs), ~2.1GHz
             Memory: 8192MB RAM

Display Devices:
          Card name: Intel(R) HD Graphics
          Chip type: Intel(R) HD Graphics Family
           DAC type: Internal
         Device Key: Enum\PCI\VEN_8086&DEV_0106&SUBSYS_397517AA&REV_09
     Display Memory: 1696 MB
   Dedicated Memory: 64 MB
      Shared Memory: 1632 MB
       Current Mode: 1366 x 768 (32 bit) (60Hz)
       Monitor Name: Generic PnP Monitor
         Monitor Id: LEN40B0
        Native Mode: 1366 x 768(p) (60.000Hz)
        Driver Name: igdumd64.dll,igd10umd64.dll,igd10umd64.dll,igdumd32,
                     igd10umd32,igd10umd32
Driver File Version: 9.17.0010.4229 (English)
     Driver Version: 9.17.10.4229
        DDI Version: 10.1
       Driver Model: WDDM 1.1

DirectX Features
    DirectX Version: DirectX 11
DX Setup Parameters: Not found
   User DPI Setting: Using System DPI
 System DPI Setting: 96 DPI (100 percent)
    DWM DPI Scaling: Disabled
 DirectDraw Acceleration: Enabled
   Direct3D Acceleration: Enabled
AGP Texture Acceleration: Enabled
  Driver Attributes: Final Retail
   Driver Date/Size: 6/1/2015 21:01:14, 12814752 bytes
        WHQL Logo'd: Yes
  Device Identifier: {D7B78E66-4246-11CF-9F64-7F19B7C2C435}
          Vendor ID: 0x8086
          Device ID: 0x0106
          SubSys ID: 0x397517AA
        Revision ID: 0x0009
 Driver Strong Name: oem24.inf:Intel.Mfg.NTamd64:iSNBM0:9.17.10.4229:
                     pci\ven_8086&dev_0106
     Rank Of Driver: 00E02001
        Video Accel: ModeMPEG2_A ModeMPEG2_C ModeWMV9_C ModeVC1_C 

Software:
     Sabaki v0.51.1  same results on both portable and installed,
                     32 and 64 bit versions
                 OS: Windows 7 Home Premium 64-bit (6.1, Build 7601)
                     Service Pack 1

(BtW, this is after a full update of Win 7.  Sabaki didn't work BEFORE
the update either, so my attempt to fix the problem this way failed.)

As to a log file - where could I find one?
Sabaki does not appear to have this option in the drop down menus,
and the program does not crash, it just sneers at me with a blank screen.
Thanks in advance for any suggestions.

**Comment by yishn** [comment]
Sabaki does not have log files, you can however [activate debug mode](https://github.com/SabakiHQ/Sabaki/blob/master/docs/guides/debugging.md) and open the developer tools to find out if there's anything wrong. You can also try disabling hardware acceleration by setting `app.enable_hardware_acceleration` to `false` in your `settings.json`. ([Where to find `settings.json`](https://github.com/SabakiHQ/Sabaki/blob/master/docs/guides/debugging.md))

**Comment by JedzieGoat** [comment]
@yishn Thanks, disabling the hardware acceleration did the trick.  This was after a few re-installs and Avast telling me that there was a virus - there wasn't.  Debug did not do a lot for me, but I discovered that Sabaki installs in more than one directory - the one I specified and C:\...\AppData\Roaming\Sabaki\.  That is where I found settings.json and tinkered with it until the program ran.  Thanks again!
